file_reader_code.R

```diff
@@ -0,0 +1,66 @@
+if (!require("pacman")) install.packages("pacman")
+library(pacman)
+pacman::p_load(dplyr, readxl, data.table, tidyr)
+
+##Function to paste together filepaths
```

Comment:
Nice use of a function; possibly might want to improve documentation of this, e.g. explain what arguments do (could look into roxygen2 package?)